metaflow/plugins/kfp/kfp.py

```diff
@@ -461,8 +462,8 @@ def _step_cli(self, node: DAGNode, task_id: str, user_code_retries: int) -> str:
             step.append(f"--split-index ${SPLIT_INDEX_ENV_NAME}")
         if self.tags:
             step.extend("--tag %s" % tag for tag in self.tags)
-        if self.namespace:
-            step.append("--namespace %s" % self.namespace)
+        if self.kfp_namespace:
+            step.append("--kfp-namespace %s" % self.kfp_namespace)
```

Comment:
This is calling the `step` command, not the `kfp` CLI command,
see here: https://github.com/zillow/metaflow/blob/yunw/pod_customization/metaflow/cli.py#L387-L387

best to revert this.

Reply:
When reverted it'd need to be the MF namespace.

Reply:
yeah make sense. will revert